main.go

```diff
@@ -46,19 +46,23 @@ func run() {
 	cobrautil.RegisterHTTPServerFlags(flags, "proxy", "proxy", ":8080", true)
 	flags.StringSlice("proxy-cors-allowed-origins", []string{"*"}, "allowed origins for CORS requests")
 
-	flags.Bool("proxy-auth-client-id", true, "Keycloak auth client ID")
+	flags.String("proxy-auth-tenant", "", "Keycloak auth tenant")
+	viper.BindPFlag("proxy-auth-tenant", flags.Lookup("proxy-auth-tenant"))
+	viper.BindEnv("proxy-auth-tenant", "PROXY_AUTH_TENANT")
+
```

Comment:
We are taking values from the environment here that we're using in HTTP requests; we should probably perform some minimal validation (e.g., only allow alphanumerics + `_`).